torchvision/transforms/functional_tensor.py

```diff
@@ -219,3 +220,40 @@ def ten_crop(img, size, vertical_flip=False):
 def _blend(img1, img2, ratio):
     bound = 1 if img1.dtype.is_floating_point else 255
     return (ratio * img1 + (1 - ratio) * img2).clamp(0, bound).to(img1.dtype)
+
+
+def resize(img, size, interpolation="bilinear"):
```

Comment:
This is something that I still need to figure out: should we use the old PIL-based `interpolation` with int, or the torch-based one? I'm not sure.